src/models/ontology_mapping_engine.py

```diff
@@ -0,0 +1,128 @@
+from src.models import ontology_mapper_st as oms
+from src.models import ontology_mapper_lm as oml
+from OmicsMLRepoHarmonizer.src.models import ontology_mapper_rag as ollm
+import src.CustomLogger.custom_logger
+import pandas as pd
+import numpy as np
+from thefuzz import fuzz
+
+logger = src.CustomLogger.custom_logger.CustomLogger()
+
+
+
+class OntoMapEngine():
+    def __init__(self, 
+                 method:str, 
+                 query:list[str], 
+                 corpus:list[str],
+                 cura_map:dict, 
+                 topk:int=5, 
+                 yaml_path:str='method_model.yaml', 
+                 om_strategy='st') -> None:
+        """
+        This class is responsible for initializing the OntoMapEngine object.
+
+        Args:
```

Comment:
Documentations on `cura_map` and `om_strategy` (including available options, i.e., `lm`,`st`, and `llm` - what they stand for?) arguments.

Reply:
Added documentation on om_strategy options